config/settings/rollover.yml

```diff
@@ -0,0 +1,8 @@
+dfe_signin:
+  secret: please_change_me # Override with SETTINGS__DFE_SIGNIN__SECRET
+  base_url: https://s121d04-rollover-publish-as.azurewebsites.net
+manage_backend:
+  base_url: https://s121d04-rollover-ttapi-as.azurewebsites.net
+environment:
+  label: "QA"
+  selector_name: "qa"
```

Comment:
are you sure `QA`?

Reply:
It's the same as what we did for `research`, so I _think_ that's correct. I'll double-check.

Reply:
Let's give it a go and see what happens, it'd be nice if it worked.